src/tests/stats/test_distribution_normal.fypp

```diff
@@ -0,0 +1,235 @@
+
+#:include "common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+program test_distribution_normal
+    use stdlib_kinds
+    use stdlib_error, only : check
```

Comment:
Might be worth to switch to using test-drive's `check` function for tests

Reply:
Don't know what is test-drive. Where is it located?

Reply:
See  e.g. [this example](https://github.com/fortran-lang/stdlib/blob/master/src/tests/stats/test_mean.fypp).
However, since the results of the different functions depend on the state of different global variables, it might not work until [this PR](https://github.com/fortran-lang/test-drive/issues/10) (or another similar) is merged.

for the moment, I propose to keep them as they are and to change them in another PR.